pymode/libs/six.py

```diff
@@ -235,7 +235,7 @@ class _MovedItems(_LazyModule):
     MovedAttribute("intern", "__builtin__", "sys"),
     MovedAttribute("map", "itertools", "builtins", "imap", "map"),
     MovedAttribute("range", "__builtin__", "builtins", "xrange", "range"),
-    MovedAttribute("reload_module", "__builtin__", "imp", "reload"),
+    MovedAttribute("reload_module", "__builtin__", "importlib", "reload"),
```

Comment:
Updating `six.py` from upstream would probably make more sense. It only uses importlib in python 3.4+: https://bitbucket.org/gutworth/six/src/92e1c746e1abfbdb94705b821bc93766083efc54/six.py?at=default&fileviewer=file-view-default#six.py-245